SampleApp/SearchPinsViewController.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  SearchPinsViewController.swift
+//  ios-sdk
+//
+//  Created by Matt Smollinger on 1/6/17.
+//  Copyright © 2017 Mapzen. All rights reserved.
+//
+
+import UIKit
+import Pelias
+
+class SearchPinsViewController: MapViewController, UITextFieldDelegate {
+
+  @IBOutlet weak var searchField: UITextField!
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    self.loadScene("scene.yaml", apiKey: "mapzen-2qQR7SX")
```

Comment:
Good idea. We should create an issue to centralize API key management in the SDK (if it does not already exits).

I would be in favor of adding that issue to the beta milestone as its not really required to get early feedback on the core functionality of the SDK which is our main goal for alpha.